release-notes/opensearch-performance-analyzer.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,13 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch 2.11.1
+
+### Infrastructure
+* Set Autopublish to true in Jenkins publish for performance-analyzer-commons repo [#45](https://github.com/opensearch-project/performance-analyzer-commons/pull/45)
```

Comment:
This does not concern PA in anyway right? I believe we can remove it? More related to PA-commons repo

Reply:
We don't have dedicated release notes for rca and commons repo, they are all included in the release notes in PA. You can check previous release notes for more information.

Reply:
You can make the call as maintainers but PA-commons follows is an independent component with its own release cycle and versioning. Would recommend to add release notes for that component so when users go and check what was new in PA-commons 1.2.0 that was used in this release, it should be there. For example: common-utils new features wont be included in dependent component release notes, etc.